README.md

```diff
@@ -23,11 +23,37 @@ Usage
 -----
 
 ```
-$ ./check_url.py example.csv
+$ ./check_url.py ./list_url.csv
```

Comment:
Why with a missing example file? Why with a `./`?

Reply:
File was renamed.

By default, expected file is `list_utl.csv` (see code source) . Therefore I've renamed example.csv to list_url.csv.

But as you wish.